### PR TITLE
WEB-9570 pattern library blog feed

### PR DESCRIPTION
**Pull request by Wmiranda1**

### CODE REVIEW

File: components/layout/blog-view.hbs [hunk 1]
```diff
@@ -0,0 +1,226 @@
+<div class="section--bg-white blog-feed">
```
Comment (fienen) [inline components/layout/blog-view.hbs:1]: Doesn't look like you're actually referencing the `blog-feed` class anywhere.

File: scss/partials/_cards.scss [hunk 2]
```diff
@@ -102,5 +111,10 @@
     .content-block__title {
       @extend .a;
     }
+    img {
+      outline: none;
+      opacity: 0.95;
```
Comment (nnedevn) [inline scss/partials/_cards.scss:116]: Just FYI, the space between the image and the author is likely to get flagged by design. 
image

File: components/layout/blog-view.hbs [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+    <a href="#" class="content-block--clickable blog-content">
+        <div class="grid-col-content__two featured">
+            <div class="content-block">
```
Comment (fienen) [inline components/layout/blog-view.hbs:4]: This feels... weird? On line 1 you've declared "this is a 'content block' that's modified as 'clickable'." Then this (and below, like line 9) says 'I'm redeclaring the stuff inside as a regular 'content-block.'" It feels very anti-patterny. A content block should be all the content, not sections inside of it. That would make them block__element level items.

File: components/layout/blog-view.hbs [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+                <img class="content-block__image" src="https://www.fillmurray.com/g/545/747"
+                    role="presentation" alt="" />
+            </div><!-- .content-block -->
+
+            <div class="content-block">
+                <h1 class="content-block__title anchor-primary">
+                    4 ways to keep your talent during the Turnover Tsunami.
+                </h1>
+
+                <div class="blog-author-date">
```
Comment (fienen) [inline components/layout/blog-view.hbs:14]: This also feels a little out of convention. Here and below on 23, you're declaring these as two very specific blocks, even though they're really elements of the blog block. I'm kinda feeling like maybe what we need is a `content-block__blog-post` or something like that at the top. Then everything else flows as `content-block__blog-author" etc. Or we just create a new block `blog-post` that extends entirely off `content-block`.

File: components/layout/blog-view.hbs [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+                </div><!-- .content-block -->
+            </a><!-- .content-block--clickable -->
+        </div><!-- .blog-content -->
+
+    </div> <!-- .two-col-content -->
+
+
+    <div class="grid-col-content__three">
+
+        <div class="blog-content">
```
Comment (fienen) [inline components/layout/blog-view.hbs:90]: Seeing this as just a wrapper to the content-block that follows makes me feel more strongly that we need a hard-fork variant of `content-block` that's specific. Either as its own block, or a clear modifier separate from --clickable.

File: scss/partials/_cards.scss (hunk 2 above)
Comment (Wmiranda1) [reply scss/partials/_cards.scss:116]: Gotcha, I messed up here the date needs to go underneath the title.

File: components/layout/blog-view.hbs [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+            <a href="#" class="content-block--clickable">
+                <div class="content-block">
+                    <img class="content-block__image" src="https://www.fillmurray.com/g/268/160"
+                        role="presentation" alt="" />
+                </div><!-- .content-block -->
```
Comment (fienen) [inline components/layout/blog-view.hbs:95]: Same comment about content-blocks inside content-blocks. But also, I think we'll want to mark this up as a <picture>+<srcset> to get the right setup for the image scaling.

File: scss/partials/_blog.scss [hunk 7]
```diff
@@ -0,0 +1,41 @@
+.grid-col-content {
```
Comment (fienen) [inline scss/partials/_blog.scss:1]: This definitely belongs in the utility partial.

File: scss/partials/_cards.scss [hunk 8]
```diff
@@ -102,5 +111,10 @@
     .content-block__title {
       @extend .a;
     }
+    img {
```
Comment (fienen) [inline scss/partials/_cards.scss:114]: As a BEM standard, we should never call out root element selectors (outside of our base partial), always classes. The only exception might be with the blog, if we're needing to do some convention breaking to support the user entered WYSIWYG content that won't have classes coded in.

**Review by fienen — APPROVED** [review]

**Pull request merged**